spring-web/src/main/java/com/webcohesion/enunciate/modules/spring_web/api/impl/ParameterImpl.java

```diff
@@ -30,6 +30,9 @@ public String getName() {
 
   @Override
   public String getDescription() {
+    if (param instanceof ExplicitRequestParameter) {
+	    return ((ExplicitRequestParameter) this.param).getDocValue();
```

Comment:
In my opinion it will be better to have one method which will return proper value from different implementations.
Basic one can return value as in line 36, and in ExplicitRequestParameter class you can use current getDocValue() implementation (if you will decice to declare getDocValue() in base class).